personalitytest/static/js/src/personalitytest-view.js

```diff
@@ -12,16 +12,53 @@ function PersonalityTestXBlockStudent(runtime, element) {
         return tmp;
     }
 
-    function paragraphize(str) {
+    function paragraphize (str) {
         var tmp = str.split('\n');
-        var buff = document.createElement('div');
+        var buff = $('<div />');
         tmp.forEach(function (item) {
-            var p = document.createElement('p');
-            p.appendChild(document.createTextNode(item));
-            buff.appendChild(p);
+            var p = $('<p />');
+            p.text(item);
+            buff.append(p);
         });
         return buff;
     }
+    function addQuestion (question, list, listCounter, studentAnswers) {
+        var questionInList = $('<li />');
+        var divQuestion = $('<div />')
+            .addClass('personality-test-question');
+
+        var select = $('<select />');
+        var opt = $('<option />')
+            .text('')
+            .val('');
+
+        select.append(opt);
+        var studentAnswer = '';
+
+        if (studentAnswers !== '') {
+            studentAnswer = getAnswersValue(studentAnswers, question['id']);
+        }
+        question['answers'].forEach(function (answer) {
+            var option = $('<option />')
+                .text(answer.answer)
+                .val(question['id']);
+
+            if (studentAnswers !== '' && studentAnswer === answer.answer) {
+                option.selected = true;
+            }
```

Comment:
No, selected should not be set as an attribute.

To change the selected value of a select, use `.val()`.